src/v3/src/components/Checkbox/Checkbox.tsx

```diff
@@ -10,121 +10,70 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-import { useOdysseyDesignTokens } from '@okta/odyssey-react-mui';
-import {
-  Box,
-  Checkbox as CheckboxMui,
-  FormControl,
-  FormControlLabel,
-  FormHelperText,
-} from '@okta/odyssey-react-mui-legacy';
-import { Fragment, h } from 'preact';
+import { Checkbox as OdyCheckbox, CheckboxGroup } from '@okta/odyssey-react-mui';
+import { h } from 'preact';
 
 import { useWidgetContext } from '../../contexts';
-import { useAutoFocus, useValue } from '../../hooks';
+import { useAutoFocus, useHtmlContentParser, useValue } from '../../hooks';
 import {
   ChangeEvent,
   UISchemaElementComponent,
   UISchemaElementComponentWithValidationProps,
 } from '../../types';
-import { getTranslationInfo } from '../../util';
-import FieldLevelMessageContainer from '../FieldLevelMessageContainer';
+import { buildFieldLevelErrorMessages, getTranslationInfo, wrapInTranslateNo } from '../../util';
 import { withFormValidationState } from '../hocs';
 
 const Checkbox: UISchemaElementComponent<UISchemaElementComponentWithValidationProps> = ({
   uischema,
   errors,
   handleChange,
   handleBlur,
-  describedByIds,
 }) => {
   const value = useValue(uischema);
   const { loading } = useWidgetContext();
 
   const {
     options: { inputMeta: { name, mutable } },
     focus,
-    noTranslate,
     translations = [],
-    showAsterisk,
   } = uischema;
   const isReadOnly = mutable === false;
+  const checkboxId = `${name}-checkbox`;
```

Comment:
We set the field name as the ID for the `CheckboxGroup` since that's the field that gets associated with aria values and then create this checkbox ID to pass to the underlying `Checkbox` component